workflow-manager/main.go

```diff
@@ -76,29 +77,42 @@ var (
 
 func fail(format string, args ...interface{}) {
 	workflowManagerLastFailure.SetToCurrentTime()
-	log.Fatalf(format, args...)
+	log.Fatal().Msgf(format, args...)
+}
+
+func prepareLogger() {
+	zerolog.LevelFieldName = "severity"
+	zerolog.TimestampFieldName = "timestamp"
+	zerolog.TimeFieldFormat = time.RFC3339Nano
+	zerolog.SetGlobalLevel(zerolog.TraceLevel)
```

Comment:
Trace is too verbose to be the default, but it'd be nice to be able to turn up logging when we need to for debugging. Can you set the default to `Info`, and provide a command line flag for log level? That said if we never log anything above `Info` then we can just turn down the default and skip the command line flag for now.